crates/core/src/sql/kind.rs

```diff
@@ -29,6 +29,8 @@ pub enum Kind {
 	Point,
 	String,
 	Uuid,
+	#[revision(start = 2)]
+	Regex,
```

Comment:
Is there a reason not to just add this to the end of the enum? AFAIK order does not matter for kinds